diesel/src/sqlite/connection/mod.rs

```diff
@@ -54,6 +54,10 @@ impl Connection for SqliteConnection {
     type Backend = Sqlite;
     type TransactionManager = AnsiTransactionManager;
 
+    /// Establish a connection to `database_url`.
+    ///
+    /// If the file `database_url` does not exist, this method will
+    /// try to create a new file and then establish a connection to it.
```

Comment:
```suggestion
    /// If the database does not exist, this method will
    /// try to create a database and then establish a connection to it.
```

As sqlite does not only allow file based databases (but also in memory databases) be a bit more generic here about what is created and how it work.